src/azure-cli/HISTORY.rst

```diff
@@ -74,6 +74,10 @@ Release History
 * Add new parameters `--enable-delete-retention` and `--delete-retention-days` to support managing delete retention policy for storage account blob-service-properties.
 
 
+**SQL**
```

Comment:
Please rearrange the release note in the right position.

Reply:
Fixed